language/en/info_acp_ideas.php

```diff
@@ -22,8 +22,10 @@
 	// ACP module
 	'ACP_PHPBB_IDEAS'			=> 'phpBB Ideas',
 	'ACP_PHPBB_IDEAS_SETTINGS'	=> 'Ideas settings',
+	'ACP_PHPBB_IDEAS_SETUP_UTILITIES'	=> 'Ideas setup utilities',
```

Comment:
This should go in the other lang file as this is not needed globally in the ACP.